apps/dolly-frontend/src/main/js/src/components/fagsystem/pensjon/visning/PoppMiljoeinfo.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { PoppVisningMiljoeinfo } from '~/components/fagsystem/pensjon/visning/PoppVisningMiljoeinfo'
+import { useAsync } from 'react-use'
+import { DollyApi } from '~/service/Api'
+import Loading from '~/components/ui/loading/Loading'
+import styled from 'styled-components'
+
+type PoppMiljoeinfoTypes = {
+	ident: string
+	bestilteMiljoer: Array<string>
+}
+
+const Miljoeinfo = styled.div`
+	margin-bottom: 10px;
+	display: flex;
+	flex-wrap: wrap;
+
+	p {
+		margin: 0;
+	}
+`
+
+export const PoppMiljoeinfo = ({ ident, bestilteMiljoer }: PoppMiljoeinfoTypes) => {
+	const miljoer = useAsync(async () => {
+		return DollyApi.getPoppMiljoer()
```

Comment:
Kunne gjort om denne til SWR hvis du har lyst til å parprogrammere det for neste PR på dette, veldig enkelt å sette opp 😄

Reply:
Lurt, tenker jeg merger inn det jeg har nå, også tar vi det på neste oppgave 🤓 Sier ifra når jeg kommer så langt 😎